### ISSUE TITLE
Add rich text option 

### ISSUE THREAD

**Issue by rakearyo** [soupslurpr/BeauTyXT#130]
Hello! 

I’m going to ask a question. I don’t think this is the right place, but I don’t have matrix or other social media. So, is there a possibility that you will add in future something like “super note” in Standard Notes ( a tool that let you change font, font size, justify the text and so on) with the possibility to export in pdf ? 

The reason I’m asking for this: I’m currently using Google Docs because for my personal needs is the best choice; however, I find out that super note mode in sn is great, but you can’t export in pdf. So, I thought maybe you could add something like this in you app. I hope I made myself clear.

Also, I just want to express my gratitude to your project. I think is really good! I’m not demanding anything here, I know this is a FOSS project and as such you’re probably building it in your free time without any payback. I’m just asking and the answer “I don’t have time/resources” is perfectly fine! :)

**Comment by soupslurpr** [comment]
Hi, this is actually supported with Typst might wanna try it out, the docs are at https://typst.app/docs/
:)

**Comment by rakearyo** [comment]
Oh my, thank you! I just tried the function and I think is amazing. You’re really cool. :)

also, I just wanted to tell you that I just tried your new app “AppVerifier” and…wow. I searched for an application like that for a long time. I love you apps because they are so simple and yet so powerful. (Just on thing: “policy and license” seems to be set incorrectly, maybe is just my phone though (I’m using a Samsung galaxy A30s with android 11)
![IMG_0422](https://github.com/soupslurpr/BeauTyXT/assets/149927000/c1af74e3-4728-4e10-9b85-ed13b300f6c8)
